### PR TITLE
Feature/test base special cases

### PR DESCRIPTION
**Pull request by fitodic**
I've updated the `setup.py`, added the `setup.cfg` file and bumped the version.
The testing of `unload/base.py` is complete.

### CODE REVIEW

**Comment by fitodic** [comment]
Ready for review.

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 setup(
     name='django-unload',
-    version='0.1',
+    version='0.2',
     url="https://github.com/Styria-Digital/django-unload",
+    author='Styria Digital Services',
     description='Remove unused template tags',
```
Comment (mislavcimpersak) [inline setup.py]: please change to "Remove unused Django template tags"

Comment (fitodic) [reply setup.py]: What do you say about: "Remove unused custom Django template tags and filters"?

Comment (mislavcimpersak) [reply setup.py]: sounds good :)

**Comment by fitodic** [comment]
Description updated.

**Pull request merged**